package.json

```diff
@@ -55,7 +55,8 @@
     "prettier": "^2.8.4",
     "supertest": "^6.3.3",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.5"
+    "typescript": "^4.6.4",
```

Comment:
Why the downgrade?

Reply:
whoops, I messed up merging the package.json with main, probably from that